tests/blueprints/networking/psc-glb-and-armor/test_plan.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 Google LLC
```

Comment:
Can you use the new style tests?

I know they're not documented (my bad) but they're much easier. Here's what you have to do: 
- Add an hcl block code at the end of the README. Similar to [this one.](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/052e310418a51dc676bc03fc39d6e6dd7764040d/blueprints/apigee/hybrid-gke/README.md#test) 
- Update the tftest directive with your own module and resource counts
- Ignore the inventory part, or if you really want I can show you how to create it.